src/registry_schemas/schemas/business.json

```diff
@@ -180,6 +185,12 @@
             "$ref": "#/definitions/complianceWarning"
           }
         },
+        "warnings": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/complianceWarning"
```

Comment:
please update L68 and L191 to be `warning` instead of `complianceWarning`

Reply:
Also please remove `complianceWarnings` block above. 

```
"complianceWarnings": {
    "type": "array",
    "items": {
      "$ref": "#/definitions/complianceWarning"
    }
  },
```

Reply:
I changed `complianceWarning` to `warning` in my latest commit.